src/site/_includes/macros/post-card.njk

```diff
@@ -1,51 +1,60 @@
-  @property {string} [className] - A space separated list of classes to apply.
+  @typedef {string} title - The title of the card, like a show's name or a post's title.
+  @typedef {string} subhead - A description or short summary.
+  @typedef {string} url - The url for the page the card is previewing.
+  @typedef {string|null} thumbnail - An relevant image to be displayed in the card.
+  @typedef {string|null} alt - Alt text for image. If no thumbnail is provided, this can be null.
+  @typedef {string[]|null} authors - An array of author ids for the authors who wrote the post.
+  @typedef {string[]|null} tags - An array of tag ids for the tags relevant to the post.
+  @typedef {Date|null} date - Date for when a post was created.
+  @typedef {Date|null} updated - Date for when a post was updated.
+  @typedef {boolean|null} isAuthor - Conditional if the card is for an author. This requires a different image.
+  @typedef {boolean|null} isDraft - Conditional if the card is for a draft post. This puts a pill on the card that says "DRAFT".
 #}
-{% macro postCard(post, featured=false, className = '') %}
+{% macro postCard(title, subhead, url, thumbnail, alt, authors, tags, date, updated, isAuthor, isDraft) %}
 
-{% set url = post.url %}
-{% set data = post.data %}
-{% set thumbnail = data.thumbnail or data.hero or null %}
-{% set alt = data.alt or '' %}
-{% set isDraft = 'w-card--draft' if data.draft %}
-{% set authors = data.authors or [] %}
+<article class="card">
+  {% if isDraft %}
+    <div class="card__header cluster flex-justify-end">
+      <p class="pill">DRAFT</p>
+    </div>
+  {% endif %}
 
-<div class="w-card {{className}} {{isDraft}}" role="listitem">
-  <article class="w-card-base {{ 'w-card-base--featured' if featured }} {{ 'w-card-base__with--cover' if thumbnail }}">
-    {% if thumbnail %}
-      <div class="w-card-base__cover">
-      {#
-        We intentionally hide the thumbnail image from screen readers because it appears out of context
-        from the rest of the article content and may be confusing. Since the link for the article is duplicated
-        in the heading down below, this might be an OK tradeoff.
-      #}
-        <a href="{{url}}" class="w-card-base__link" tabindex="-1" aria-hidden="true">
-          <figure class="w-card-base__figure">
-            {% Img src=thumbnail, alt=alt, width="354", height="240", class="w-card-base__image", params={fit: 'crop', h: '240', w: '354'} %}
-          </figure>
-        </a>
-      </div>
-    {% endif %}
-    <div class="w-card-base__blurb">
-      <a href="{{url}}" class="w-card-base__link">
-        <h2 class="{{ 'w-card-base__headline--with-image' if thumbnail else 'w-card-base__headline' }}">{{data.title}}</h2>
+  {% if thumbnail %}
+    {% if isAuthor %}
+      <a href="{{ url }}" aria-hidden="true" class="avatar gap-inline-auto" data-size="large">
+        {% Img src=thumbnail, alt=title, width="192", height="192", params={fit: 'crop', h: '192', w: '192'} %}
```

Comment:
The alt should be a descriptor of the image, not the title of the post. E.g. post title might be ""Web Vitals Metrics" but the image alt would be "A calculator with some numbers displayed on it".

Reply:
That was a mistake, I copied the img tag from the i18n-card which used titles. I'm fixing that now, nice catch!